Makefile

```diff
@@ -167,7 +167,7 @@ endif
 
 .depdirs: $(foreach SDIR, $(CONFIGURED_APPS), $(SDIR)_depend)
 
-.depend: context Makefile .depdirs
```

Comment:
@liuguo09 
Are there any side effects of removing this dependency?, because this is a top level Makefile.

Reply:
@liuguo09 

>So remove context dependency for apps
>depend to make sure context build only one time for apps.

I understood and confirmed that context build for apps is done only once.